lib/core/services/navigation/navigation_service_impl.dart

```diff
@@ -32,7 +32,7 @@ class NavigationServiceImpl implements NavigationService {
   }
 
   @override
-  bool pop({returnValue}) {
-    return navigatorKey.currentState.pop(returnValue);
+  void pop({returnValue}) {
```

Comment:
This should remain bool, because pop still returns a bool.

Reply:
As you can see in the image above, pop doesn't return bool anymore.